adapters/handlers/graphql/local/aggregate/aggregate.go

```diff
@@ -124,6 +124,10 @@ func classField(class *models.Class, description string,
 			},
 			"nearVector": nearVectorArgument(class.Class),
 			"nearObject": nearObjectArgument(class.Class),
+			"objectLimit": &graphql.ArgumentConfig{
+				Description: descriptions.First,
```

Comment:
is this a valid description?

Reply:
I think so, it is the same description used for `limit`